ui/src/app/configuration/consul-configs-boolean/consul-configs-boolean.component.ts

```diff
@@ -21,14 +21,12 @@ import { StreampipesPeContainerConifgs } from '../shared/streampipes-pe-containe
 import { ConfigurationService } from '../shared/configuration.service';
 
 @Component({
-    selector: 'consul-configs-boolean',
+    selector: 'sp-consul-configs-boolean',
```

Comment:
Oh, my bad, I forgot updating the file names. I'll update the PR with the changes.

Reply:
I'm  nor sure if this is best practive
So it would be best to wait for a conformation from @tenthe @dominikriemer or @smlabt 

But apart from that: thanks a lot for your great work again 🙏🏼

Reply:
Thanks @aamirxshaikh for the awesome work!
@bossenti The `sp` prefix in the selector only is fine for me and we don't need to prefix all files with sp as there is no danger of conflicts when reusing the component.

Reply:
Alright, thanks for clarifying 🙏 
But do we need to watch for adapting references as well that might be affected by the renaming?

Reply:
Can you check that again @aamirxshaikh, please? 
For example in `consul-configs.component.html` the reference is not refactored

Reply:
Just to clarify @bossenti, we need to make sure the reference to component is not updated with `sp` prefix right?

Reply:
We need to check for every selector that has been prefixed with `sp-` that all usages of the component use the new name/selector.
`consul-configs.component.html` is one example where this isn't updated

Reply:
Alright. Got it. So, In general we need to search for the component usages in updated files and update to the new selector name i.e prefixed with `sp`.

Reply:
I have updated the found usages to the selector to the new name. Shall I update the pr?

Reply:
That would be great 🤩